markdown/chapters/meandeck/matchups.md

```diff
@@ -31,16 +31,6 @@ fast piles.
 ::accordion[Cards to keep in mind]{path=matchups/delver.ur.cards}
 ::accordion[Gameplay]{path=matchups/delver.ur.gameplay}
 
```

Comment:
Can replace with Jeskai Days Undoing? Swap Ragavan line for Normal Jeskai shit?

Reply:
I'd rather do that in a separate MR (this one is strictly about purging Ragavan content and will be easier to search for/revert to)